### PR TITLE
Build /download/mediatek page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Don't merge:** Release date postponed, waiting for a official release date, possibly around March
** Blocked** Waiting on 3 links
- Builds /download/mediatek page based off of [copydoc](https://docs.google.com/document/d/1Tm3cVVkWkzriQQ7kgbgKfKoeGZaew1M5OMbpLPSDGIc/edit#heading=h.142hynaix65t)

## QA
- Demo at https://ubuntu-com-12110.demos.haus/
- Go to /download/mediatek  and compare against [copydoc](https://docs.google.com/document/d/1Tm3cVVkWkzriQQ7kgbgKfKoeGZaew1M5OMbpLPSDGIc/edit#heading=h.142hynaix65t)
- Check the download links download the correct image

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5990

### CODE REVIEW

File: templates/download/mediatek.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>Download Ubuntu Server</h2>
+      <h3 class="p-heading--4">Ubuntu Server 22.04 LTS</h4>
+      <p>The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
+      <p><a href="#" class="p-button--positive">Download 64-bit</a></p>
```
Comment (minkyngkm) [inline templates/download/mediatek.html:39]: The link is not working...

```suggestion
      <p><a href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-22.04/iot-limerick-kria-classic-desktop-2204-x06-20220614-78.img.xz" class="p-button--positive">Download 64-bit</a></p>
```

File: templates/download/mediatek.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        height="2160",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <h3 class="p-heading--4">First time installing Ubuntu on MediaTek?</h3>
+    <p>Please check the <a href="https://www.mediatek.com/aiot/products/mediatek-genio/1200">MediaTek Genio page</a> for more information.</p>
```
Comment (minkyngkm) [inline templates/download/mediatek.html:59]: The link works but there's nothing in the page...
image

File: templates/download/mediatek.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    <div class="p-notification--information is-inline">
+      <div class="p-notification__content">
+        <p class="p-notification__title">Note:</p>
+        <p class="p-notification__message"> Please ensure that your MediaTek Genio platform has the latest boot FW. FW updates and instructions are available on the <a href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-22.04/iot-limerick-kria-classic-desktop-2204-x06-20220614-78.img.xz">link to be provided</a>.</p>
```
Comment (minkyngkm) [inline templates/download/mediatek.html:63]: The text "link to be provided" needs to be updated..

File: templates/download/mediatek.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <h3 class="p-heading--4">First time installing Ubuntu on MediaTek?</h3>
+    <p>Please check the <a href="https://www.mediatek.com/aiot/products/mediatek-genio/1200">MediaTek Genio page</a> for more information.</p>
+    <div class="p-notification--information is-inline">
+      <div class="p-notification__content">
+        <p class="p-notification__title">Note:</p>
+        <p class="p-notification__message"> Please ensure that your MediaTek Genio platform has the latest boot FW. FW updates and instructions are available on the <a href="#">link to be provided</a>.</p>
```
Comment (minkyngkm) [inline templates/download/mediatek.html:99]:
```suggestion
        <p class="p-notification__message"> Please ensure that your MediaTek Genio platform has the latest boot FW. FW updates and instructions are available on the <a href="https://www.mediatek.com/aiot/products/mediatek-genio/1200">link to be provided</a>.</p>
```

**Comment by anthonydillon** [comment]
@petesfrench who can we chase to get this unstuck?

**Comment by anthonydillon** [comment]
@petesfrench can you close or merge this yet?

**Comment by petesfrench** [comment]
Spoke with Felicia and we decided as this is outdated, she will make a new request when the content is actually ready. Closing

**Comment by petesfrench** [comment]
Closing as we are building the page in the new style, so wont need this PR

**Pull request closed without merging**